internal/ingress/controller/store/store.go

```diff
@@ -891,6 +891,10 @@ func (s *k8sStore) setConfig(cmap *corev1.ConfigMap) {
 	s.backendConfigMu.Lock()
 	defer s.backendConfigMu.Unlock()
 
+	if cmap == nil {
+		return
+	}
```

Comment:
Why is this necessary now?

Reply:
Without it we get an exception in the tests (this happens only after updating the dependencies)